docs/ext/bridge/api.rst

```diff
@@ -151,3 +151,8 @@ BridgeContext Subclasses
 
 .. autoclass:: discord.ext.bridge.BridgeExtContext
     :members:
+
+.. attributetable:: discord.ext.bridge.Context
+
+.. autoclass:: discord.ext.bridge.Context
+    :members:
```

Comment:
Welp

Reply:
Well it's not a class so that doesn't work
Ideally it should have cleaner documentation, something like 
```
Alias of ``:class:`BridgeExtContext` | :class:`BridgeApplicationContext``` for typing convenience.
```

Reply:
How's this?

<img width="783" alt="Screenshot 2023-06-11 at 4 32 45 PM" src="https://github.com/Pycord-Development/pycord/assets/92863779/58dbe134-a99b-47a9-b4d2-1a22d3423f4b">